plaso/cli/image_export_tool.py

```diff
@@ -760,9 +760,8 @@ def ParseOptions(self, options):
 
     self._ParseFilterOptions(options)
 
-    if (getattr(options, 'no_vss', False) or
```

Comment:
Do you confirm we want to drop the check against no_vss?

Reply:
Yes, for context see: https://github.com/log2timeline/plaso/issues/2757